jenkins/swift-functional-tests/10-install-ring.sh

```diff
@@ -1,12 +1,28 @@
 #!/bin/bash -xue
 
+function get_AllowEncodedSlashes {
```

Comment:
Yes but our driver (in its 0.3 branch) won't encode any slash, so I agree with Nicolas.
Are you sure this make any difference on test results in 0.3 to have that ?